WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2716,7 +2716,6 @@ static GSourceFuncs _handlerIntervention =
             TRACE_L1("%s", URL.c_str());
 
             urlValue(URL);
-            notifyUrlLoadResult(URL, Core::ERROR_NONE);
```

Comment:
Why is it moved after clients notification?
Was there any problem calling notifyUrlLoadResult() in this place?